### PR TITLE
Adding new insight for reply-to domain age <=30d and does not match sender domain

### PR DESCRIPTION
**Pull request by morriscode**
image

### CODE REVIEW

**Review by ianthiel — APPROVED** [review]

File: insights/headers/reply_to_mismatch_new.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "New reply-to domain (<=30d) and sender mismatch"
+type: "query"
+source: |
+  map(filter(headers.reply_to,
+    length(headers.reply_to) > 0
+    and any(headers.reply_to, .email.domain.root_domain != sender.email.domain.root_domain
+    and  beta.whois(.email.domain).days_old <= 30)
```
Comment (rw-access) [inline insights/headers/reply_to_mismatch_new.yml:7]:
```suggestion
    and beta.whois(.email.domain).days_old <= 30)
```

File: insights/headers/reply_to_mismatch_new.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: "New reply-to domain (<=30d) and sender mismatch"
+type: "query"
+source: |
+  map(filter(headers.reply_to,
+    length(headers.reply_to) > 0
```
Comment (rw-access) [inline insights/headers/reply_to_mismatch_new.yml:5]: this check doesn't do anything for the filter

Comment (morriscode) [reply insights/headers/reply_to_mismatch_new.yml:5]: Roger that will pull and validate! thx

Comment (morriscode) [reply insights/headers/reply_to_mismatch_new.yml]: Tested worked great.

File: insights/headers/reply_to_mismatch_new.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: "New reply-to domain (<=30d) and sender mismatch"
+type: "query"
+source: |
+  map(filter(headers.reply_to,
+    any(headers.reply_to, .email.domain.root_domain != sender.email.domain.root_domain
+    and beta.whois(.email.domain).days_old <= 30)
+  ), .email.domain.root_domain)
```
Comment (rw-access) [inline insights/headers/reply_to_mismatch_new.yml:7]:
```suggestion
  map(filter(headers.reply_to,
    .email.domain.root_domain != sender.email.domain.root_domain
    and beta.whois(.email.domain).days_old <= 30
  ), .email.domain.root_domain)
```

**Review by rw-access — APPROVED** [review]

**Pull request merged**